lib/firefox/webdriver/index.js

```diff
@@ -134,6 +134,29 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
     }
   }
 
+  if (firefoxConfig.disableSafeBrowsing) {
```

Comment:
Good solution to this dilemma.
This is on me to figure out the performance impact but the safebrowsing tables also inform resource scheduling in Necko - priority, tailing, throttling.